Packs/FeedGCPWhitelist/Integrations/FeedGCPWhitelist/FeedGCPWhitelist.yml

```diff
@@ -0,0 +1,100 @@
+category: Authentication
+commonfields:
+  id: GCP Whitelist
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  defaultvalue: Good
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: ""
+  name: feedExpirationPolicy
+  defaultvalue: indicatorType
+  type: 17
+  required: false
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+- defaultvalue: '20160'
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '30'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Use the Google Cloud Platform whitelist integration to get indicators
+  from the feed.
+display: GCP Whitelist
+name: GCP Whitelist
+script:
+  commands:
+  - arguments:
+    - default: false
+      isArray: false
+      name: limit
+      description: The maximum number of results to return.  Default value is Unlimited.
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets indicators from the feed.
+    execution: false
+    name: gcp-whitelist-get-indicators
+    outputs:
+    - contextPath: hello
+      description: Should be Hello **something** here
+      type: String
+  dockerimage: gcp2:latest
```

Comment:
You need to check if there is an existing docker image in our docker images [repo](https://github.com/demisto/dockerfiles) that would be appropriate for use with your integration.  If you are searching for a specific package and want to see if it is available in a published docker image, you can use the [used_packages.csv](https://github.com/demisto/dockerfiles/blob/repository-info/used_packages.csv) file which contains a listing of packages and all docker images using that package. If there isn't, you can follow the directions there to create one for use with the integration.